src/routes/workshop-a11y-fundamentals.svelte

```diff
@@ -203,174 +550,348 @@
-        <span aria-hidden="true">#3:</span>
-        Screen readers
-      </strong>
-      <p class="p">Meaningful content, landmarks, forms and common design patterns.</p>
-    </li>
-  </ol>
-
-  <h2 class="f-title subtitle">Workshop Dynamics</h2>
-  <p class="p">The workshop consists in a set of challenges with detailed resources.</p>
-  <p class="p">
-    Each challenge introduces a new topic, followed by a hands-on exercise for you to apply the
-    concepts learned.
+  <h3 class="t-title asH3">
+    Workshop dynamics
+  </h3>
+  <p class="t-p">
+    In each challenge, I’ll introduce you to a new topic in a simplified way, along with <strong>detailed resources</strong>. 
+  </p>
+  <p class="t-p">
+    Then, you'll have a <strong>hands-on exercise</strong> for you to apply the concepts learned. You can solve it by yourself or by collaborating with a group of 2-3 people.
   </p>
-  <p class="p">
-    After each exercise, we'll go through the solution together and I will clarify any question that
-    you may have.
+  <p class="t-p">
+    Afterwards, we go through the <strong>solution</strong> together, and I’ll <strong>clarify the questions</strong> that you might have.
   </p>
 
-  <h2 class="f-title subtitle">Pre-requisites</h2>
+  <div class="t-card t-process">
+    <ul class="t-processFlow">
+      <li class="t-processFlowIx">theory</li> 
+      <li class="t-processFlowIx">practice</li> 
+      <li class="t-processFlowIx">review</li> 
+      <li class="t-processFlowIx">clarify</li> 
+      <li class="t-processFlowIx">repeat</li> 
+    </ul>
+    <p>An interactive way of learning together.</p>
+  </div>
+
 
-  <ul class="list">
-    <li>Understanding of HTML and CSS (Javascript is a bonus);</li>
-    <li>A modern browser installed: Chrome or Firefox are recommended;</li>
+  <h3 class="t-title asH3">
+    Who is this workshop for?
+  </h3>
+  <p class="t-p">
+    Web developers are the main audience, but if you are a designer,
+    or a QA expert then this workshop will be valuable for you as well. 
+  </p>
+  <p class="t-p">
+    The topics will be explained with beginners in mind and it gets more advanced as we go through it.
+  </p>
+  <p class="t-p">
+    Even if you already have some experience into A11Y, you can look at this workshop as a way to solidify your knowledge and fill any existing gap.
+  </p>
+
+  <h3 class="t-title asH3">
+    Pre-requisites
+  </h3>
+  <ul class="t-list">
+    <li>Comfortable with HTML and CSS. Basics of JavaScript is a bonus;</li>
+    <li>A <strong>modern browser</strong> installed: Chrome or Firefox are recommended;</li>
     <li>
-      A good internet connection with
+      A <strong>good internet connection</strong> with
       <a class="u-link" href="http://zoom.com/" rel="noreferrer">Zoom</a>
       installed for the video call;
     </li>
+    <li>
+      A webcam is optional. Although I highly appreacite seeing you while I’m speaking, I want to be respectful of your own space.
```

Comment:
A webcam is optional - although I highly appreciate seeing you while I’m speaking, I want to be respectful of your boundaries.